pkg/controller/perconaservermongodb/psmdb_controller.go

```diff
@@ -529,11 +537,13 @@ func (r *ReconcilePerconaServerMongoDB) checkConfiguration(cr *api.PerconaServer
 	return nil
 }
 
-func (r *ReconcilePerconaServerMongoDB) safeDownscale(cr *api.PerconaServerMongoDB) error {
+// return whether need to downscale
```

Comment:
this comments seems a bit unclear on what the function does, how about?

```suggestion
// safeDownscale ensures replica set pods downscaled one by one and returns true if a downscale is in progress
```